qlib/workflow/exp.py

```diff
@@ -176,8 +176,6 @@ def __init__(self, id, name, uri):
         self.client = mlflow.tracking.MlflowClient(tracking_uri=self._uri)
 
     def start(self, recorder_name=None):
-        # set the active experiment

```

Comment:
Why this line is deleted?

Reply:
"set_experiment" will create mlruns, which we do not want to be created at here.